hedera-mirror-rosetta/build/Dockerfile

```
@@ -25,56 +27,31 @@ FROM ubuntu:20.04 as runner
-
-# Install PostgreSQL 9.6, supervisor, git and openjdk-11
 ARG DEBIAN_FRONTEND=noninteractive
-RUN apt-get update && apt-get install -y postgresql-9.6 postgresql-client-9.6 postgresql-contrib-9.6 supervisor openjdk-11-jdk-headless curl
-
-USER root
+ENV PG_VERSION=13
+RUN apt-get update \
+    && apt-get install -y ca-certificates gnupg lsb-release \
+    && apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8 \
+    && echo "deb https://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list \
+    && apt-get update \
+    && apt-get install -y postgresql-${PG_VERSION} postgresql-client-${PG_VERSION} supervisor openjdk-11-jre-headless \
+    && rm -rf /var/lib/apt/lists/*
+
+COPY --from=builder /app /app
+# Create users for importer and rosetta
+RUN useradd -s /bin/bash importer && useradd -s /bin/bash rosetta
+
+ENV PATH="/usr/lib/postgresql/${PG_VERSION}/bin:${PATH}"
+ENV PGCONF="/etc/postgresql/${PG_VERSION}/main"
+ENV PGDATA="/var/lib/postgresql/${PG_VERSION}/main"
 
-# Create Volume PostgreSQL directory and Change default PostgreSQL directory
-RUN mkdir -p /data/db
-RUN chown postgres /data/db
-RUN chmod 700 /data/db
-RUN mv /var/lib/postgresql/9.6/main /data/db/main
-RUN ln -s /data/db/main /var/lib/postgresql/9.6/main
-
-# ---------------------------  Supervisord  --------------------------- #
-
-COPY hedera-mirror-importer/src/main/resources/db/scripts/init.sh /scripts/init.sh
 USER postgres
-ENV PATH="/usr/lib/postgresql/9.6/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
-ENV PGCONF="/etc/postgresql/9.6/main"
-ENV PGDATA="/var/lib/postgresql/9.6/main"
-
+RUN cp /app/build/pg_hba.conf ${PGCONF}/ && cp /app/build/postgresql.conf ${PGCONF}/conf.d
 # Init db script
-RUN /etc/init.d/postgresql start && /scripts/init.sh
-
-RUN echo "listen_addresses='*'" >> "${PGCONF}/postgresql.conf"
-# Allow PG Admin access
-RUN echo "host    all             all             172.17.0.1/16           trust" >> "${PGCONF}/pg_hba.conf"
+RUN /etc/init.d/postgresql start && /app/scripts/init.sh && /etc/init.d/postgresql stop
 
 USER root
-
-# Copy the Rosetta Executable from the Rosetta Builder stage
-WORKDIR /var/rosetta
-COPY --from=rosetta-builder /hedera-mirror-rosetta/hedera-mirror-rosetta .
-
-# Copy the Importer Jar and Config from the Java-Builder stage
-WORKDIR /var/importer
-COPY --from=java-builder /hedera-mirror-node/hedera-mirror-importer/target/hedera-mirror-importer-*exec.jar ./hedera-mirror-importer.jar
-# Enable persisting non-fee transfers to reconstruct the original transfer list
-ENV HEDERA_MIRROR_IMPORTER_PARSER_RECORD_ENTITY_PERSIST_NONFEETRANSFERS=true
-# Disable Redis because gRPC process is not run within this execution
-ENV HEDERA_MIRROR_IMPORTER_PARSER_RECORD_ENTITY_REDIS_ENABLED=false
-
-COPY hedera-mirror-rosetta/build /build
-WORKDIR /build
+WORKDIR /app/build
```

I think it makes more sense for the app to not run out of a build directory
```suggestion
WORKDIR /app
```

---

I can copy the entrypoint script and the supervisor config files out to /app then.